src/TouristInfoCards.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+const TouristInfoCards = () => {
```

Comment:
This component renders all 3 tourist info cards. But if I wanted to show just 1 of the cards in another place in the app, I'd have to copy this code to where I wanted it.

Instead if we made a component that only rendered 1 card, and we passed it some props for the image, url and name then we can re-use the component where-ever we want.